content/en/docs/concepts/services-networking/service.md

```diff
@@ -71,14 +71,17 @@ balancer in between your application and the backend Pods.
 
 ## Defining a Service
 
-A Service in Kubernetes is a REST object, similar to a Pod.  Like all of the
-REST objects, you can `POST` a Service definition to the API server to create
-a new instance.
+A Service in Kubernetes is a REST object (just as a Pod or a ConfigMap is an object
+that you can access as a resource in the Kubernetes HTTP API.
+You can use the HTTP API to create a new Service instance, to apply changes to an
+existing Service, or to request deletion.
```

Comment:
I'm wary of comparing Service (singular) with Pods and ConfigMaps (both plural). I also want to avoid starting the sentence with “Service” because I want the emphasis that the initial capital provides.

Reply:
Maybe "similar to a Pod or a ConfigMap," ?